leaderboard.py

```diff
@@ -0,0 +1,131 @@
+# Import the PostgreSQL library
+import psycopg2
+
+# Import database connection settings
+from config import config
+
+# Connect to TreeTracker Database and return connection object
+def connect():
+    
+    conn = None
+    try:
+        # read connection parameters
+        params = config()
+
+        # connect to the PostgreSQL server
+        conn = psycopg2.connect(**params)
+        
```

Comment:
@CareyAltgilbers we can not create the connection this way on the airflow framework, we can not read db config file on the cloud, so that why we have to do like this:

https://github.com/Greenstand/treetracker-airflow-dags/blob/3204621ea8058d970a9cf63522dcb04ae625479a/capture_export_dag.py#L55-L94

So the way to develop and release is: making the db connecting agnostic, just pass the connection object into the function, so on the development, you can use any way you can to get the connection settings, but on cloud, we must use the API by airflow to get it.

This is an example of how to get db connection locally: https://github.com/Greenstand/treetracker-airflow-dags/blob/3204621ea8058d970a9cf63522dcb04ae625479a/lib/capture_export_test.py#L8-L20

the env variable is a normal OS env var, you just need to make sure it's set correctly